cub/cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -62,6 +63,38 @@
 
 CUB_NAMESPACE_BEGIN
 
+template <typename MaxPolicyT,
+          typename InputIteratorT,
+          typename OutputIteratorT,
+          typename ScanOpT,
+          typename InitValueT,
+          typename OffsetT,
+          typename AccumT,
+          bool ForceInclusive>
+struct DeviceScanKernelSource
```

Comment:
important: we probably don't want this struct as part of our public API. Let's wrap it into `detail` namespace.

Reply:
Fixed - I placed it in `detail::scan` namespace.